docs/changes.rst

```diff
@@ -232,7 +232,7 @@ Version 1.0.10rc1 (2017-01-29)
 Bug fixes:
 
 - borg serve: fix transmission data loss of pipe writes, #1268
-  This affects only the cygwin platform (not Linux, *BSD, OS X).
+  This affects only the cygwin platform (not Linux, BSD, OS X).
```

Comment:
Looks like some changes should go into 1.0-maint, while others can't go there. So it's a bit of a mixup here.